### DISCUSSION TITLE
Who voiced the character of 🫏 in Shrek?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#950] [Galaxy Trivia] [answered]
**Bonus:** What was the accent that he was original going to use in the 🎥?

**Comment by peterbaccaro** [comment, marked as answer]
The character of Donkey in the Shrek film series is voiced by actor **Eddie Murphy**. Eddie Murphy's energetic and comedic performance brought the character of Donkey to life with his distinct voice and humorous delivery. He provided the voice for Donkey in all four Shrek films: "Shrek" (2001), "Shrek 2" (2004), "Shrek the Third" (2007), and "Shrek Forever After" (2010), as well as in various Shrek spin-offs and specials.

In the early stages of development for the film Shrek, the character of Donkey was **originally intended to have a more traditional, stereotypical Eddie Murphy voice**. However, during the recording sessions, Eddie Murphy and the filmmakers decided to take a different approach. They opted for a more contemporary, urban voice for Donkey, closer to Eddie Murphy's natural speaking voice rather than a specific accent.

The decision was made to make Donkey's voice more relatable and contemporary, allowing Eddie Murphy to bring his own comedic style and personality to the character. This change proved successful, and Eddie Murphy's portrayal of Donkey became one of the iconic elements of the Shrek franchise.
